src/main/java/Group15/ExerciseDetailsView.java

```diff
@@ -0,0 +1,106 @@
+package Group15;
+
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.geometry.Pos;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+
+public class ExerciseDetailsView {
+
+    private static String exerciseTitle = "Bench Press";
+    private static String exerciseDescription = "This is a description of bench press. It is a compound exercise that works multiple muscle groups including the chest, shoulders, and triceps. Just press Bro!";
+    private static String exerciseType = "Compound";
+    private static String exerciseBodyPart = "Chest, Shoulders, Triceps";
+    private static String exerciseEquipment = "Bench, Barbell";
+    private static String exerciseDifficulty = "Intermediate";
+
+    public static Scene createScene() {
+        VBox layout = new VBox();
+        layout.setSpacing(20);
+        layout.setAlignment(Pos.CENTER);
+
+        Pane exerciseImagePane = createExerciseImagePane();
+        layout.getChildren().add(exerciseImagePane);
+
+        Pane exerciseInfoPane = createExerciseInfoPane();
+        layout.getChildren().add(exerciseInfoPane);
+
+        Pane buttonPane = createButtonPane();
+        layout.getChildren().add(buttonPane);
+
+        return new Scene(layout, 1000, 800);
+    }
+
+    private static Pane createExerciseImagePane() {
+        VBox imagePane = new VBox();
+        imagePane.setAlignment(Pos.CENTER);
+        imagePane.setPrefSize(640, 400);
+
+        Image exerciseImage = new Image("file:src/main/resources/benchPress.png");
```

Comment:
This should be added to the Exercise class

Reply:
Also, we should really think about whether we want pictures, it would take a lot of manual labour

Reply:
We will keep it for now and then we can discuss it later if we want it to be a part of the exercise class or we will scrap the idea entirely